go.mod

```diff
@@ -0,0 +1,71 @@
+	github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96
+	github.com/emicklei/go-restful v2.6.0+incompatible
+	github.com/emicklei/go-restful-openapi v0.10.0
+	github.com/fromanirh/okdutil v0.0.1
+	github.com/ghodss/yaml v0.0.0-20150909031657-73d445a93680
+	github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8
+	github.com/go-kit/kit v0.0.0-20160928073314-fe6fe28ba0d5
+	github.com/go-logfmt/logfmt v0.4.0
+	github.com/go-openapi/analysis v0.17.2
+	github.com/go-openapi/errors v0.17.2
+	github.com/go-openapi/jsonpointer v0.19.0
+	github.com/go-openapi/jsonreference v0.19.0
+	github.com/go-openapi/loads v0.17.2
+	github.com/go-openapi/runtime v0.17.2
+	github.com/go-openapi/spec v0.17.2
+	github.com/go-openapi/strfmt v0.17.0
+	github.com/go-openapi/swag v0.19.0
+	github.com/go-openapi/validate v0.18.0
+	github.com/go-stack/stack v1.8.0
+	github.com/gogo/protobuf v0.0.0-20170330071051-c0656edd0d9e
+	github.com/golang/glog v0.0.0-20141105023935-44145f04b68c
+	github.com/golang/mock v0.0.0-20190517145838-140ae90f29ea
+	github.com/golang/protobuf v1.2.0
+	github.com/google/btree v0.0.0-20160524151835-7d79101e329e
+	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367
+	github.com/google/uuid v1.0.0
+	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d
+	github.com/gorilla/websocket v0.0.0-20180228210902-0647012449a1
+	github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7
+	github.com/hashicorp/golang-lru v0.0.0-20160207214719-a0d98a5f2880
+	github.com/imdario/mergo v0.3.5
+	github.com/json-iterator/go v0.0.0-20180612202835-f2b4162afba3
+	github.com/k8snetworkplumbingwg/network-attachment-definition-client v0.0.0-20181121151021-386d141f4c94 // indirect
+	github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515
+	github.com/mailru/easyjson v0.0.0-20180823135443-60711f1a8329
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
+	github.com/modern-go/reflect2 v0.0.0-20180320133207-05fbef0ca5da
+	github.com/onsi/ginkgo v1.8.0
+	github.com/onsi/gomega v1.5.0
+	github.com/openshift/api v0.0.0-20190524175055-ed2bab04cd2e
+	github.com/openshift/client-go v0.0.0-20180830153425-431ec9a26e50
+	github.com/pborman/uuid v1.2.0
+	github.com/peterbourgon/diskv v2.0.1+incompatible
+	github.com/spf13/pflag v1.0.1
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
+	golang.org/x/net v0.0.0-20190311183353-d8887717615a
+	golang.org/x/oauth2 v0.0.0-20170412232759-a6bd8cefa181
+	golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a
+	golang.org/x/text v0.3.0
+	golang.org/x/time v0.0.0-20161028155119-f51c12702a4d
+	google.golang.org/appengine v0.0.0-20190515044707-311d3c5cf937
+	gopkg.in/inf.v0 v0.9.0
+	gopkg.in/yaml.v2 v2.2.1
+	k8s.io/api v0.0.0-20181128191700-6db15a15d2d3
+	k8s.io/apiextensions-apiserver v0.0.0-20181217193801-853f76028711
+	k8s.io/apimachinery v0.0.0-20181128191346-49ce2735e507
+	k8s.io/client-go v0.0.0-20181205110444-5e6a3d4e34f6
```

Comment:
feel free to update this to make sure it matches  kubevirt v0.17.0. In a future patch we will bump the dep to kubevirt v0.18.z anyway